flink-clients/src/main/java/org/apache/flink/client/deployment/executors/AbstractSessionClusterExecutor.java

```diff
@@ -80,7 +80,7 @@ public CompletableFuture<JobClient> execute(
                     clusterDescriptor.retrieve(clusterID);
             ClusterClient<ClusterID> clusterClient = clusterClientProvider.getClusterClient();
             return clusterClient
-                    .submitJob(jobGraph)
+                    .submitJob(jobGraph, pipeline)
```

Comment:
@PatrickRen  How should we send lineagegraph into the cluster client without changing API?

Reply:
I think that you are saying that the original submitJob with one parameter still works. As well as the new method.

Reply:
Yes, but there is a concern about the size of lineage info if put into job graph.